src/Benchmarks.ServerJob/Scenario.cs

```diff
@@ -3,9 +3,21 @@
 
 namespace Benchmarks.ServerJob
 {
+    /// <summary>
+    /// The scenario is used both as a commandline argument to the Benchmarks process and also to compute
+    /// the path portion of the URL. These definitions should match those in the Benchmarks process.
+    /// 
+    /// Use [ScenarioPath(...)] when the path portion of the URL doesn't match the scenario name exactly.
+    /// </summary>
     public enum Scenario
     {
         Plaintext,
-        Json
+        Json,
+
+        [ScenarioPath("/mvc/plaintext")]
+        MvcPlaintext,
+
+        [ScenarioPath("/mvc/json")]
+        MvcJson,
```

Comment:
@mikeharder - needed to add this (or something like it) to be able to work with multiple path segments. If you don't like this approach let's discuss